daprdocs/content/en/getting-started/quickstarts/bindings-quickstart.md

```diff
@@ -11,10 +11,1080 @@ Let's take a look at Dapr's [Bindings building block]({{< ref bindings >}}). Usi
 - Trigger your app with events coming in from external systems.
 - Interface with external systems.
 
-In this Quickstart, you will schedule a batch script to run every 10 seconds using an input [Cron]({{< ref cron.md >}}) binding. The script processes a JSON file and outputs data to a SQL database using the [PostgreSQL]({{< ref postgresql.md >}}) Dapr binding.
+In this Quickstart, you will schedule a batch script to run every 10 seconds using an input [Cron]({{< ref cron.md >}}) binding. The script processes a JSON file and outputs data to a SQL database using the [PostgreSQL]({{< ref postgresql.md >}}) Dapr binding. You can run this quickstart by either:
+- [Running all applications in this sample simultaneously with the Multi-App Run template file]({{< ref "#run-using-multi-app-run" >}}), or
+- [Running one application at a time]({{< ref "#run-one-application-at-a-time" >}})
 
 <img src="/images/bindings-quickstart/bindings-quickstart.png" width=800 style="padding-bottom:15px;">
 
+## Run using Multi-App Run
+
+{{% alert title="Note" color="primary" %}}
```

Comment:
We can remove this preview note, since this will not be preview in v1.12 and also not really needed here anyway

Reply:
I do not see the multi-app run template use example in this binding example. There are only individuals run examples. Where is this added? We should also place the multi-app run first.